event_model/__init__.py

```diff
@@ -2211,3 +2212,47 @@ def default(self, obj):
                 return obj.item()
             return obj.tolist()
         return json.JSONEncoder.default(self, obj)
+
+def discover_handlers(entrypoint_group_name='databroker.handlers',
+                      skip_failures=True):
+    """
+    Discover handlers via entrypoints.
+    Parameters
```

Comment:
Needs a blank line. If the original in databroker is missing one too that should be fixed there as well.